### DISCUSSION TITLE
DAW -> Google Hangouts

### DISCUSSION THREAD

**Discussion by FortunateSmith** [ExistentialAudio/BlackHole#89] [General]
I've been teaching music lessons via Hangouts during the pandemic and I'm curious if there's a way to play audio from my computer through Hangouts. I'm using Black Hole to create an aggregate output and am able to get microphones and instruments to play through Hangouts from Logic Pro X. However if I drag an mp3 into the project and play it, while I can hear it on my end, my student cannot hear it on the other end. Is there a good/relatively easy workaround for this?

**Comment by devinroth** [comment]
My guess is you aren’t routing the audio out on that track properly. What I do is output the track to my to blackhole and use a send to send it to my speakers (via the aggregate device of course). You need to make sure you are sending to BlackHole ch1-2. 

Hope that makes sense.
